### PR TITLE
Add missing ending single quote on subscription-manager command

### PR DESCRIPTION
**Pull request by themoosman**
#### What is this PR About?
Fix a minor error with the subscription-manager command on line 833 as it is missing an ending quote.

#### How should we test or review this PR?
Verify the updated command.

#### Is there a relevant Trello card or Github issue open for this?
No, found the issue while working with a client.

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this @sabre1041 @etsauer

### CODE REVIEW

File: playbooks/installation/index.adoc [hunk 1]
```diff
@@ -830,7 +830,7 @@ cat /etc/yum.repos.d/ ...
 ==== Subscribing directly to Red Hat
 
 ----
-ansible -i c1-ocp.myorg.com/hosts OSEv3 -a 'subscription-manager register --username bob@acme.com --password='mypassword'
+ansible -i c1-ocp.myorg.com/hosts OSEv3 -a 'subscription-manager register --username bob@acme.com --password='mypassword''
```
Comment (etsauer) [inline playbooks/installation/index.adoc]: Looks like those quotes need to be escaped.

Comment (themoosman) [reply playbooks/installation/index.adoc]: Should we just remove the unnecessary quotes around the password parameter?  That would simplify the command.  @etsauer

Comment (etsauer) [reply playbooks/installation/index.adoc]: @themoosman as long as the command works, I think thats the better way

**Comment by lpsantil** [comment]
@themoosman Thanks for catching this.  I saw this a few months ago (whoops!) and it totally slipped my mind to create the PR.

**Comment by themoosman** [comment]
@etsauer Remove the single quotes around the password on line 833.  The single quotes aren't necessary and I removed them to avoid confusion and copy/paste errors.

**Pull request merged**